frontend/src/Components/IndicatorBars/PlantDetailBar.js

```diff
@@ -1,28 +1,68 @@
 import PropTypes from 'prop-types'
+import { useState } from 'react'
 import styled from 'styled-components'
+import { SettingsIcon } from '../Icons'
+import SettingCareIntervalModal from '../Modals/SettingCareIntervalModal'
 
 export default function PlantDetailBar({ daysLeft, weeksLeft }) {
   PlantDetailBar.propTypes = {
     daysLeft: PropTypes.number.isRequired,
     weeksLeft: PropTypes.number.isRequired,
   }
```

Comment:
```suggestion
  PlantDetailBar.propTypes = {
    daysLeft: PropTypes.number.isRequired,
    weeksLeft: PropTypes.number.isRequired,
  }
 
 export default function PlantDetailBar({ daysLeft, weeksLeft }) {
```
No need to redefine PropTypes on every update.

Reply:
Haha, yeah I have to change it for every file at one point... It's on my list for the last week 😅